.github/workflows/cd.yml

```
@@ -0,0 +1,32 @@
+name: cd
+on:
+  workflow_run:
+    workflows: ["ci"]
+    branches-ignore: [main]
```

```suggestion
    branches-ignore: [*]
```
You probably need to ignore for all branches and only run on tag